README.md

```diff
@@ -34,14 +34,15 @@ You need to use Parallels. ([Parallels Manual](./docs/parallels.md))
 ```console
 git clone https://github.com/lf-edge/eden.git
 cd eden
+make clean
 make build
 eden config add default
 eden setup
 eden start
 eden eve onboard
 eden status
 make build-tests
-eden test /tests/workflow
+eden test ./tests/docker
```

Comment:
wait, so why is this docker now all of a sudden? the README is describing the entire workflow, right? @mydatascience